### DISCUSSION TITLE
blog/domain-name-information

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#20] [Announcements]
# blog/domain-name-information

我为何更换.xyz域名？godaddy如何坐地起价？我的网站的新域名意义是什么？

https://rercel.com/blog/domain-name-information

**Comment by tsurumi-yizhou** [comment]
这实际上让我想起了猪多期刊出版商；科学知识是属于全人类的，但刊载科学知识的文章不是。

**Comment by iwenyi** [comment]
GoDaddy 是注册商，如果不满意的话可以换另外一家，就像是国内的阿里云和新网那样。而 CNNIC 是 .cn 注册局，不论你的 .cn 域名在哪个注册商注册的，它都可以管理你的域名。.com 域名注册局是 Verisign ，GoDaddy 只是注册商，不要混淆了两者的概念。

另外再提一下，你说的有价值的域名很多在很久以前就被米农注册了。域名交易是一个规模很大的产业，米农也有很多囤有价值的 .cn 域名，你可以搜搜一些含义好的 .cn 域名，很多会显示出售页。这与哪个域名后缀没有太大的关系，主流的域名后缀大部分有价值的域名基本上在米农手里。

至于 .xyz 这样的 New gTLD 域名，它们的注册局基本上是商业公司，肯定会到处推广做营销活动。买它的人大部分冲着首年便宜去的，自然而然滥用问题很严重。想要长久使用的话，还是像 .com .net .org 这样传统 gTLD 域名靠谱，历史悠久久经考验，不易受各种因素的影响，这就是为什么大部分公司喜欢用 .com。ccTLD 容易受所属的国家或地区的政策影响，网上不喜欢 CNNIC 的人经历过 .cn 域名回收事件。

**Reply by iwenyi** [reply]
补充一下，gTLD 和 New gTLD 域名除了 .org 的注册局 Public Interest Registry (PIR) 是非营利性组织，其它的注册局都是商业公司。

**Reply by Pathsis** [reply]
谢谢指教！希望以后不再更换域名了。

**Reply by iwenyi** [reply]
不用客气。如果你不考虑以后国内可访问性，用现在的域名也不错。如果你在意国内访客，最好是选一个可以备案的 TLD，哪怕是现在不备案，以后出现一些不可抗拒的问题时可以备案继续使用。可以备案的后缀名单在这里：https://domain.miit.gov.cn/

关于不可抗拒的因素，不备案的域名有时候被区别对待，即使你的网站没有不合规内容。比如说一些地方有白名单，不备案的网站无法访问，跟以前被墙了的网站待遇差不多。

别的我不说了，话题比较敏感。

**Reply by Pathsis** [reply]
确实这个问题值得考虑，谢谢提醒！我目前遇到的问题是，部署在 Vercel 上的这个网站国内移动和电信访问太慢了。不说影响读者了，连我自己访问都遭罪。但是搬回国内吧，也有许多不想面对的事情。这个问题还在考虑中。

**Reply by iwenyi** [reply]
只是对访问速度不满意，一时半会儿不想备案的话，我倒是有一个办法。

你知道云计算厂商普遍有 S3 这项服务吗？国内的云计算厂商叫做对象存储，可以用来部署静态网站。选择香港和新加坡这样离大陆近的节点，至少目前来说速度还是不错的，而且不用备案，价格也很便宜。

我看你网站有议论时事的文章，备案了的话应该不允许写这种内容吧。备案挺麻烦，而且有很多限制，一发现哪里不合适就会要求整改。有一些地方还要求公安备案，尤其是开评论需要备交互式，这就是为什么很多备案了的博客不开评论区的原因。
